R/estimate_outcomes.R

```diff
@@ -12,8 +12,8 @@
 #' @return A `<data.frame>` with the columns in `data`, and with two additional
 #' columns:
 #'
-#'  - `"known_outcomes"` for the total number of known outcomes on that day
-#' of the outbreak, and
+#'  - `"estimated_outcomes"` for the total number of outcomes estimated to be
+#' known on that day of the outbreak, and
```

Comment:
maybe since we are changing this, to avoid any kind of confusion, it could be specified what "that day" refers to? e.g., on the day specified in column X

Reply:
Yes, changed above